Libraries/LibWeb/Layout/SVGFormattingContext.cpp

```diff
@@ -191,6 +191,10 @@ void SVGFormattingContext::run(AvailableSpace const& available_space)
     //       that we are laying out an SVG fragment of another document.
     //       Rephrase: "We are not laying out a standalone SVG document"
     //       Current browser behavior seems to guarantee initial about:blank for layout of SVG fragments.
+    // FIXME: Find another way to determine whether we're laying out a standalone SVG document.
```

Comment:
same with this fixme, squash into the first commit.

Reply:
Noted. Squashed.